_targets.R

```diff
@@ -115,35 +117,128 @@ targets_list <- list(
 
   # BUILD QUERY
   # Define list of GCMs
-  tar_target(gcm_names, c('ACCESS', 'GFDL')),#, 'CNRM', 'IPSL', 'MRI', 'MIROC')),
-  tar_target(gcm_vars, c("evspsbl", "hfss")),
-  tar_target(gcm_dates, c('1999-01-01', '1999-01-15')),
+  tar_target(gcm_names, c('ACCESS', 'GFDL', 'CNRM', 'IPSL', 'MRI', 'MIROC')),
+  tar_target(gcm_dates_df,
+             tibble(
+               projection_period = c('1980_1999', '2040_2059', '2080_2099'),
+               start_datetime = c('1980-01-01 00:00:00', '2040-01-01 00:00:00', '2080-01-01 00:00:00'),
+               # Include midnight on the final day of the time period
+               end_datetime = c('2000-01-01 00:00:00', '2060-01-01 00:00:00', '2100-01-01 00:00:00')
+             )),
+  # Gathered manually from https://cida.usgs.gov/thredds/ncss/notaro_GFDL_2040_2059/dataset.html
+  tar_target(gcm_query_vars, c("pr", "tas", "qas", "rsns", "uas", "vas")), # missing longwave radiation
```

Comment:
Just for our records, might be helpful to include what each of these corresponds to in a comment, since the abbreviations aren't super helpful.
![image](https://user-images.githubusercontent.com/54007288/148292718-241b10d9-ab9a-478e-aed7-4168c21cc7de.png)

Reply:
Oh and looks like we might be able to use that `ps` surface pressure variable to [calculate RelHum](https://github.com/USGS-R/lake-temperature-model-prep/blob/main/7_drivers_munge/src/GLM_driver_utils.R#L151) from the `qas` specific humidity variable.